### ISSUE TITLE
FlakeID Generator

### ISSUE THREAD

**Issue by burakcelebi** [hazelcast/hazelcast-csharp-client#292]
Hazelcast FlakeID Generator is not available to .NET Client.

https://docs.hazelcast.org/docs/4.0/manual/html-single/index.html#flakeidgenerator

**Comment by adhip94** [comment]
Yes, I too would like to see this added in the current version of the .NET Client 3.12.x.

**Comment by osmedd** [comment]
Is this in the plan for a 4.x client release soon?  Since it was supported in the 3.12.x client, is it a big deal to support with the 4.x server?  I see it is supported in the 4.x cpp-client...
